com.unity.shadergraph/Tests/Editor/GraphUI/GraphUITests.cs

```diff
@@ -0,0 +1,29 @@
+using System;
+using NUnit.Framework;
+using System.Collections.Generic;
+using System.IO;
+using UnityEngine.UIElements;
+using UnityEditor.GraphToolsFoundation.Overdrive;
+using UnityEngine;
+
+namespace UnityEditor.ShaderGraph.GraphUI.UnitTests
+{
+    [TestFixture]
+    class UITestFixture
```

Comment:
Will we be using this test fixture for other things?

Reply:
> Is this the icon for opening the blackboard? com.unity.shadergraph/Editor/GraphUI/GraphElements/Stylesheets/Icons/Blackboard.png

It is yup, scrounged that from the base GTF icon library

Reply:
Yup the idea is that this is a place to store any UI automated tests that we want to implement, which, despite the somewhat limited ability to automate UI stuff right now, we still do have some automated UI tests in old ShaderGraph for interaction with the Blackboard etc. that I'd like to carry over

Reply:
Okay, so this is a placeholder for now. That's fine.